src/Sentry/Laravel/Tracing/Integrations/IntegrationInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Sentry\Laravel\Tracing\Integrations;
+
+interface IntegrationInterface
+{
+    public static function supported(): bool;
```

Comment:
Not sure if the notion of being supported is an exact semantic fit here, perhaps something like `isApplicable()` would be more expressive?